### PR TITLE
Add public device type

### PR DESCRIPTION
**Pull request by tmigone**
Change-type: patch
Signed-off-by: Tomás Migone <tomas@balena.io>

---
##### Contributor checklist

- [ ] Includes tests
- [ ] Includes typings
- [ ] Includes updated documentation
- [ ] Includes updated build output

### CODE REVIEW

File: lib/types/models.ts [hunk 1]
```diff
@@ -205,6 +206,15 @@ export interface PublicOrganization {
 	handle: string;
 }
 
+export interface PublicDevice {
+	id: string;
+	latitude: string;
+	longitude: string;
+	belongs_to__application: NavigationResource<Application>;
```
Comment (iamsolankiamit) [inline lib/types/models.ts:213]: This and `is_of__device_type` is not returned by the api yet!

Comment (tmigone) [reply lib/types/models.ts:212]: Removed `id`, however `is_of__device_type` is being returned if I run:

```
await window.sdk.pine.get({
    resource: 'public_device',
    options: {  $top: 1 }
});
```

I get: 
```
[{…}]
0:
belongs_to__application: {__id: 1635297, __deferred: {…}}
is_of__device_type: {__id: 145, __deferred: {…}}
latitude: ""
longitude: ""
was_recently_online: false
```

**Review by thgreasi — COMMENTED** [review]
Looks correct, let's just add `public_device: PublicDevice` in the `ResourceTypeMap` near the top.

File: lib/types/models.ts (hunk 1 above)
Comment (thgreasi) [reply lib/types/models.ts:212]: It indeed does return them now :tada:

**Comment by tmigone** [comment]
Oops missed that, it's fixed now.

**Review by thgreasi — APPROVED** [review]

**Comment by thgreasi** [comment]
@balena-ci retest

**Pull request merged**